src/scipp/html/formatting_html.py

```diff
@@ -441,8 +441,8 @@ def _obj_repr(header_components, sections):
     sections = "".join(f"<li class='sc-section-item'>{s}</li>" for s in sections)
 
     return ("<div>"
-            f"{load_icons()}"
-            "<div class='sc-wrap'>"
+            f"{load_icons()}<style>{load_style()}</style>"
+            "<div class='sc-wrap sc-root'>"
```

Comment:
Is this always the top level element?

Reply:
I don't know, what are you referring to exactly? I just reverted this to how it was originally.

Reply:
I was thinking of nested reprs, i.e. variables inside data arrays. This would put the root class onto non-root elements.
But can this happen?

Reply:
No, it cannot (unless we want to use this for representing bins).